[DOCS] Add the "investigate in timeline" button to the alert prevalence section (#2077)
elastic/security-docs: docs/detections/alerts-ui-manage.asciidoc

The *Highlighted Fields* section displays the most relevant fields for the alert type. Use this section to inform your triage efforts as you investigate the alert.

> The *Alert prevalence* column shows the total number of alerts within the selected timeframe that have identical values. For example, an alert prevalence of 3 for the `host.name` field means three alerts with the same `host.name` value exist within the given timeframe. Alert prevalence data can help you investigate relationships with other alerts and gain more context about the event producing the alert. Click the alert prevalence count to explore the alerts in Timeline. 

Reviewer: Some minor suggestions for conciseness and flow. Adding the new sentence at the end made the paragraph feel a little long, so maybe breaking into two paras could help:
```suggestion
The *Alert prevalence* column shows the total number of alerts within the selected timeframe that have identical values. For example, an alert prevalence of 3 for `host.name` means three alerts with the same `host.name` value exist within the timeframe. 

Alert prevalence data can help you investigate relationships with other alerts and gain context about the events producing alerts. You can also click the alert prevalence count to explore the alerts in Timeline. 
```

Author: I'm on board with this. I usually try to keep all related content together, but I also like the idea of breaking paragraphs into 2-3 sentences when the sentences are on the longer side.

> The *Alert prevalence* column shows the total number of alerts within the selected timeframe that have identical values. For example, an alert with an alert prevalence of 3 for the `host.name` field means three alerts with the same `host.name` value exist within the given timeframe. Alert prevalence data can help you investigate relationships with other alerts and gain more context about the event producing the alert. Click the alert prevalence count to explore the alerts in Timeline. 

Reviewer 2: ```suggestion
The *Alert prevalence* column shows the total number of alerts within the selected timeframe that have identical values. For example, an alert prevalence of 3 for the `host.name` field means three alerts with the same `host.name` value exist within the given timeframe. Alert prevalence data can help you investigate relationships with other alerts and gain more context about the event producing the alert. Click the alert prevalence count to explore the alerts in Timeline. 
```
